src/Stratis.Bitcoin.Features.Miner/BlockDefinition.cs

```diff
@@ -604,9 +604,9 @@ private int UpdatePackagesForAdded(TxMempool.SetEntries alreadyAdded, Dictionary
         /// <summary>Update the block's header information.</summary>
         protected void UpdateBaseHeaders()
         {
+            this.block.Header.Nonce = 0;
```

Comment:
Any reason for moving this here?

Reply:
I am planning to re-use some of the nonce bits to increase datetime granularity, hence i have to re-order the assignment of values to the fields.. It will make sense later, but for now this is not a logic change.